pipelines/jobs/configurations/jdk17u_pipeline_config.groovy

```diff
@@ -240,6 +240,7 @@ class Config17 {
                                 'special.system'
                         ],
                         weekly : [
+                                'dev.external',
```

Comment:
This is on the IBM CE spec. Is this expected or is it intended to be for the CRIU or OE spec?

Reply:
This is intended